### ISSUE TITLE
Missing fs/mongo records following 4/24 watcher termination

### ISSUE THREAD

**Issue by jasmynp** [microbiomedata/nmdc_automation#722]
@samobermiller noticed a few records that completed successfully in JAWS but were not post-processed i.e., not in mongo or on the fs.  
One example listed in slack:

> There are downstream workflows from the original ReadsQC that ran and succeeded 4/26 (nmdc:wfmgas-11-g248b059.1 jaws 178288, nmdc:wfrbt-11-6b37wz63.1 jaws 178372). Results should be at nmdc:dgns-15-vyb8bp30/ , but they are not. And they are therefore also not in mongo.

I took a look at the `watcher-prod_full.log` and do not see messages like "`Processing successful job`" indicating that it started the post-process for the wf activity IDs listed above, and correspondingly, they are not in the agent.state file. These jobs were submitted on 4/24/26 shortly before an abrupt watcher termination, so I suspect this disrupted the cycle before the agent.state was updated.

@samobermiller will generate a list of missing records that need to complete their processing post-JAWS and I will restore the agent.state.  We will discuss what code improvements can be made to help minimize this failure type in the short and long term.

cc: @aclum

**Comment by jasmynp** [comment]
Processed Sam's list of 55 IDs and updated the agent.state. The watcher was restarted on 5/11 and the IDs were picked up by the watcher for post-processing.
